rosdep/python.yaml

```diff
@@ -6115,6 +6115,21 @@ python3-boto3:
     '*': ['python%{python3_pkgversion}-boto3']
     '7': null
   ubuntu: [python3-boto3]
+python3-box:
+  debian:
+    '*': [python3-box]
+    buster:
+      pip:
+        packages: [python-box]
+  fedora: [python-box]
```

Comment:
```suggestion
  fedora: [python3-box]
```

Reply:
Some context here for the record: Fedora started providing un-versioned virtual packages for Python 3 subpackages. Previously, un-versioned Python packages were assumed to be Python 2.

In an effort to reduce friction in a hypothetical Python 4 release someday and also more closely align with the actual binary package name and the binary package names on other distributions, I think that using the `python3-` prefix for Python 3 packages on Fedora and RHEL is the right thing to do even if `python-` does technically work.

Reply:
Thank you for the info!